# Array indices from 0 to max match joints from base joint to last joint
int8[] modules

# Base position
geometry_msgs/Vector3 base_pos